command-expansion-server/src/app.ts

```diff
@@ -17,15 +31,35 @@ app.use(bodyParser.json({ limit: '25mb' }));
 
 DbExpansion.init();
 const db = DbExpansion.getDb();
-const graphqlClient = new GraphQLClient(getEnv().MERLIN_GRAPHQL_URL);
 
-const logger = getLogger("app");
+const piscina = new Piscina({ filename: `${path.resolve()}/build/worker.js` });
+
+type Context = {
+  commandTypescriptDataLoader: InferredDataloader<typeof commandDictionaryTypescriptBatchLoader>,
+  activitySchemaDataLoader: InferredDataloader<typeof activitySchemaBatchLoader>,
+  simulatedActivityInstanceDataLoader: InferredDataloader<typeof simulatedActivityInstanceBatchLoader>,
+  expansionSetDataLoader: InferredDataloader<typeof expansionSetBatchLoader>,
+};
+
+app.use(async(req: Request, res: Response, _next: NextFunction) => {
+  const graphqlClient = new GraphQLClient(getEnv().MERLIN_GRAPHQL_URL);
+
+  const context: Context = {
+    commandTypescriptDataLoader: new DataLoader(commandDictionaryTypescriptBatchLoader({graphqlClient})),
+    activitySchemaDataLoader: new DataLoader(activitySchemaBatchLoader({graphqlClient})),
+    simulatedActivityInstanceDataLoader: new DataLoader(simulatedActivityInstanceBatchLoader({graphqlClient})),
+    expansionSetDataLoader: new DataLoader(expansionSetBatchLoader({graphqlClient})),
+  };
+
+  res.locals.context = context;
+  return _next();
```

Comment:
Minor cleanup - remove the leading underscore here and in the function parameters -> the leading underscore typically indicates and unused parameter (which this is not)